packages/cron/test/storage.spec.js

```diff
@@ -0,0 +1,139 @@
+    'QmNvTjdqEPjZVWCvRWsFJA1vK7TTw1g9JP6we1WBJTRADM'
+  ]
+  const uploadSize = 439804651110
+  const pinnedSize = 109951162800
+  const largeFileSize = 3199023255550
+  const pins = [
+    {
+      status: 'Pinning',
+      location: {
+        peerId: '12D3KooWFe387JFDpgNEVCP5ARut7gRkX7YuJCXMStpkq714ziK6',
+        peerName: 'web3-storage-sv15',
+        region: 'region'
+      }
+    },
+    {
+      status: 'Pinned',
+      location: {
+        peerId: '12D3KooWFe387JFDpgNEVCP5ARut7gRkX7YuJCXMStpkq714ziK7',
+        peerName: 'web3-storage-sv16',
+        region: 'region'
+      }
+    }
+  ]
+
+  beforeEach(async () => {
+    user1 = await createUser(dbClient, {
+      name: 'test1-name',
+      email: 'test1@email.com'
+    })
+    authKey1 = await createUserAuthKey(dbClient, Number(user1._id), {
+      name: 'test1-key'
+    })
+    await createUpload(dbClient, Number(user1._id), Number(authKey1), cids[0], {
+      dagSize: uploadSize
+    })
+    await createPsaPinRequest(dbClient, authKey1, cidsPinned[0], {
+      dagSize: pinnedSize,
+      pins
+    })
+    await createPsaPinRequest(dbClient, authKey1, cidsPinned[1], {
+      dagSize: pinnedSize,
+      pins
+    })
+
+    user2 = await createUser(dbClient, {
+      name: 'test2-name',
+      email: 'test2@email.com'
+    })
+    authKey2 = await createUserAuthKey(dbClient, Number(user2._id), {
+      name: 'test2-key'
+    })
+    await createUpload(dbClient, Number(user2._id), Number(authKey2), cids[0], {
+      dagSize: uploadSize
+    })
+    await createUpload(dbClient, Number(user2._id), Number(authKey2), cids[1], {
+      dagSize: uploadSize
+    })
+
+    user3 = await createUser(dbClient, {
+      name: 'test3-name',
+      email: 'test3@email.com'
+    })
+    authKey3 = await createUserAuthKey(dbClient, Number(user3._id), {
+      name: 'test3-key'
+    })
+    await createUpload(dbClient, Number(user3._id), Number(authKey3), cids[0], {
+      dagSize: uploadSize
+    })
+    await createUpload(dbClient, Number(user3._id), Number(authKey3), cids[1], {
+      dagSize: Math.round(uploadSize * 1.2)
+    })
+    await createPsaPinRequest(dbClient, authKey3, cidsPinned[0], {
+      dagSize: pinnedSize,
+      pins
+    })
+
+    user4 = await createUser(dbClient, {
+      name: 'test4-name',
+      email: 'test4@email.com'
+    })
+    authKey4 = await createUserAuthKey(dbClient, Number(user4._id), {
+      name: 'test4-key'
+    })
+    await createUserTag(dbClient, Number(user4._id), {
+      tag: 'StorageLimitBytes',
+      value: '2199023255552'
+    })
+    const cid = 'bafybeibvuy3vcepqxy4plr34twv22vvxol2jjhmjxcrcvuhea5226whpsm'
+    await createUpload(dbClient, Number(user4._id), Number(authKey4), cid, {
+      dagSize: largeFileSize
+    })
+  })
+
+  it('can be executed', async () => {
+    const { stderr: emailLog1 } = await execa('./src/bin/storage.js', { env })
+    assert.match(emailLog1, /storage:checkStorageUsed 🗄 Checking users storage quotas/)
```

Comment:
Can we be testing actual data or function calls instead of stderr?

Reply:
Raised as a follow-up improvement ticket #1203

Reply:
cool